packages/docz-core/src/bundlers/webpack/dotenv.ts

```diff
@@ -0,0 +1,88 @@
+import fs from 'fs'
```

Comment:
please, move this file inside to `src/config` to be together with the `paths.ts` file...

Reply:
👍